### DISCUSSION TITLE
Export To LoRA as a saving option?

### DISCUSSION THREAD

**Discussion by CCpt5** [wkpark/sd-webui-model-mixer#50] [Ideas]
First of all, I've been following the development of your extension from the start, and you deserve a ton of credit for your dedication and what you've created. I wish you were rewarded w/ more stars on this repo as it absolutely deserves it, and clearly you understand the ins and outs of the models/coding as well as anyone. Thanks so much for your contributions here to the open source community!!

Not an important topic/subject, but I'm curious if any LoRA functionality could potentially be added to MMixer in the future? 

Personally, the feature I'm really seeking is a way to export (or technically extract) a LoRA from the current model. I do a lot of Dreambooth training and use Model Mixer to balance multiple trials etc. At the moment I use an old version of the Kohya LoRA extraction script to make a LoRA from models I've trained. Supermerger is inconsistent in its ability to properly extract them (often errors), and it's not on the actual TXT2IMG page which is one thing I love about MM.  
  
Thanks for any insight regarding general LoRA functionality ever being included in a future version. 

Continued success! (and if you have a buy me a coffee/etc lmk!!)

-C

**Comment by CCpt5** [comment]
Laugh emoji? Sorry if this was offensive - I'll delete it later.

**Reply by wkpark** [reply]
no! you don't have to delete your opinion.
sd-webui already has an integrated LoRA functionality, and we can merge LoRAs + checkpoint with a small fix.
I will add minimal function to support merged LoRAs + checkpoint in near future
(see also https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/13808 )
and as you know there are good sources for LoRA/LoCon exporter, like https://github.com/bmaltais/kohya_ss, Lora block weights and the Supermerger. (I will focus on quick testing and easy UI)

**Comment by wkpark** [comment]
initial LoRA/LyCORIS save/export support added! PR #51
